content/vault/v1.20.x/content/docs/deploy/kubernetes/csi/index.mdx

```diff
@@ -1,34 +1,38 @@
 ---
 layout: docs
-page_title: Vault CSI provider
+page_title: Vault Secrets Store CSI provider
 description: >-
-  Learn how the Vault CSI provider allows pods to consume Vault secrets using CSI volumes.
+  Learn how to consume Vault secrets using CSI volumes with the Vault provider
+  for the Secrets Store CSI driver.
 ---
 
-# Vault CSI provider
+# Vault Secrets Store CSI provider
 
-The Vault CSI Provider allows pods to consume Vault secrets using
-[CSI Secrets Store](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes.
+The Vault Secrets Store CSI provider allows pods to consume Vault secrets using
+[Secrets Store CSI](https://github.com/kubernetes-sigs/secrets-store-csi-driver) volumes. 
+Alternatively, you can deploy HashiCorp's
+[Vault Secrets Operator CSI driver](/vault/docs/platform/k8s/vso/csi) as part of your
+Vault Secrets Operator installation.
 
-The Vault CSI provider is a fully supported component of HashiCorp Vault.
+The Vault Secrets Store CSI provider is a fully supported component of HashiCorp Vault.
```

Comment:
```suggestion
```
I know this was part of the original doc, but it's extraneous. We don't document anything in our product docs that *isn't* fully supported